Utility/ARM/Update-AzureModule.ps1

```diff
@@ -28,8 +40,36 @@ Param
     [String] $AutomationResourceGroup,
 
     [Parameter(Mandatory=$True)]
-    [String] $AutomationAccount
+    [String] $AutomationAccount,
+
+    [Parameter(Mandatory=$False)]
+    [object] $ModuleVersionOverrides
     )
+
+$versionOverrides = ""
+## Try to parse module version overrides
```

Comment:
Any specific reason to use double # for comments? This makes me think these comments have a special meaning. If they don't, let's use just a single #.